src/module/ovs/ovs.rs

```diff
@@ -45,6 +46,33 @@ impl Collector for OvsCollector {
         cmd.register_module::<OvsCollectorArgs>(ModuleId::Ovs)
     }
 
+    // Check if the OvS collector can run. Some potential errors are silenced,
+    // to avoid returning an error if we can't inspect a given area for some
+    // reasons.
+    fn can_run(&self) -> Result<()> {
+        let inspector = inspect::inspector()?;
+
+        // First check the OvS kernel module is available.
+        if let Ok(kconf) = inspector.kernel.get_config_option("CONFIG_OPENVSWITCH") {
```

Comment:
OVS did maintain for some time an oot kernel module (in OVS's tree) that could be used. Not that it's very common AFAIK (plus, it was removed from OVS's tree earlier this year), but: wouldn't the presence of the module be enough?